### ISSUE TITLE
Make Community Version Available on PyPi?

### ISSUE THREAD

**Issue by edmenendez** [jet-community/django-jet#36]
Unfortunately, the old project seems to have stalled/been abandoned. I think it would be appropriate to make a django-jetnew or something like that and push that to pypi. There is a long list of PRs and fixes that the community has made available which are not being merged into the abandoned project.

I find this thread to be useful for etiquette on forking abandoned projects:

https://dev.to/ben/whats-the-proper-etiquette-for-forking-an-abandoned-repo-d5h

**Comment by SalahAdDin** [comment]
Yes, it is, and, right know, i have no time for helping with this, and the other guys left me alone. If you know who can help with this i can give permissions for all.
